packages/react/src/DomProps.ml

```diff
@@ -1659,7 +1659,14 @@ let camelcaseToKebabcase str =
   in
   str |> chars_of_string |> loop [] |> List.rev |> string_of_chars
 
-let findByName tag jsxName =
+let findByName name =
```

Comment:
Where this is being used?

Reply:
It was used on `getPropByName` for `domProps` on `ReactDom`. But removed due to this comment: https://github.com/ml-in-barcelona/server-reason-react/pull/171#discussion_r1804281202